etna/pipeline/hierarchical_pipeline.py

```diff
@@ -330,46 +333,3 @@ def _forecast_prediction_interval(
 
         finally:
             self.forecast, self.raw_forecast = self.raw_forecast, self.forecast  # type: ignore
-
```

Comment:
👍

Reply:
We have this simplification after removing `self._fit_ts`.